src/Sentry/Protocol/Metrics/DistributionMetric.cs

```diff
@@ -0,0 +1,37 @@
+using Sentry.Extensibility;
+using Sentry.Internal.Extensions;
+
+namespace Sentry.Protocol.Metrics;
+
+/// <summary>
+/// Distributions track a list of values over time in on which you can perform aggregations like max, min, avg.
+/// </summary>
+internal class DistributionMetric : Metric
+{
+    public DistributionMetric()
+    {
+        _value = new List<double>();
+    }
+
+    public DistributionMetric(string key, double value, MeasurementUnit? unit = null,
+        IDictionary<string, string>? tags = null, DateTime? timestamp = null)
+        : base(key, unit, tags, timestamp)
+    {
+        _value = new List<double>() { value };
+    }
+
+    private readonly List<double> _value;
```

Comment:
fields above constructor?

Reply:
We can. I've often been putting the fields next to the properties they back but happy to put them above the constructor.